apps/pie-docs/src/components/toast/overview/overview.md

```diff
@@ -0,0 +1,376 @@
+    items: [
+        "**Info:** Providing contextual information about the progress of an action",
+        "**Warning:** Providing non-critical information about the progress of an action",
+        "**Success:** Providing positive information about the progress of an action",
+        "**Error:** Providing critical information about the progress of an action. Please review if Toast is correct to use it might be better to use a dialogue, notification or banner",
+        "**Neutral:** Providing critical information about the progress of an action. Please review if Toast is correct to use it might be better to use a dialogue, notification or banner"
+    ]
+} %}
+
+### Icons
+
+When incorporating an icon into a Toast, it is essential to ensure that the icon clearly conveys the intended feedback. You should only use the icons built into the component and not use a different icon.
+
+{% usage {
+    do: {
+        type: usageTypes.image,
+        items: [{
+            src: "../../../assets/img/components/toast/modifiers-icons-do.svg",
+            width: "456px",
+            alt: "Two toast components with close icon button"
+        }]
+    },
+    dont: {
+        type: usageTypes.image,
+        items: [{
+            src: "../../../assets/img/components/toast/modifiers-icons-dont.svg",
+            width: "280px",
+            alt: "A toast without close icon button nor dismiss button"
+        }]
+    }
+} %}
+
+### Actions 
+
+If you are using actions, only use the pre-built button in the toast.
+
+{% usage {
+    do: {
+        type: usageTypes.image,
+        items: [{
+            src: "../../../assets/img/components/toast/modifiers-actions-do.svg",
+            width: "456px",
+            alt: "Two toast components with close icon button and dismiss button"
+        }]
+    },
+    dont: {
+        type: usageTypes.image,
+        items: [{
+            src: "../../../assets/img/components/toast/modifiers-actions-dont.svg",
+            width: "280px",
+            alt: "A toast with a wrong dismiss button variation"
+        }]
+    }
+} %}
+
+---
+
+## Sizes
+
+### Height
+
+The toast has a minimum height of 48px when displaying one line of body copy, and it should not exceed 122px in height to accommodate up to three lines of multi-line body copy.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/toast/sizes-height.svg",
+    alt: "A regular toast and a multi-line toast.",
+    width: 300
+} %}
+
+### Width
+
+The toast has a minimum width of 300px and a maximum width of 420px.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/toast/sizes-width.svg",
+    alt: "A regular toast and a multi-line toast.",
+    width: 420
+} %}
+
+---
+
+## Overflow
+
+If the body copy extends beyond one line, it should automatically wrap to multiple lines. Yet, ensure that multi-line text is truncated to a maximum of three lines. For longer text content, consider utilising a pop-up component.
```

Comment:
```suggestion
If the body copy extends beyond one line, it should automatically wrap to multiple lines. Multi-line text will be truncated so that it does not exceed three lines. For longer text content, consider utilising a pop-up component.
```
I'm not sure what is being suggested when we say "pop-up component".